hello_world/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION ${CMAKE_VERSION})
+project(HelloWorld LANGUAGES CXX)
+
+include(ExternalProject)
+include(GNUInstallDirs)
+
+add_subdirectory(src)
```

Comment:
don't `add_subdirectory` `src` and `include`, just define the library in this file.